models/sipnet/R/read.restart.SIPNET.R

```diff
@@ -4,70 +4,65 @@
 ##' 
 ##' @param outdir      output directory
 ##' @param runid       run ID
-##' @param time        year that is being read
+##' @param stop.time        year that is being read
 ##' @param settings    PEcAn settings object
-##' @param variables   variables to be extracted
+##' @param var.names   var.names to be extracted
 ##' 
 ##' @description Read Restart for SIPNET
 ##' 
 ##' @return X.vec      vector of forecasts
 ##' @export
 ##' 
-read.restart.SIPNET <- function(outdir,runid,time,settings,variables,sample_parameters=NULL){
+read.restart.SIPNET <- function(outdir,runid,stop.time,settings,var.names,params){
 
-  if(sample_parameters == TRUE){
-    load(file.path(settings$outdir, "samples.Rdata"))
-    prior.sla <- mean(ensemble.samples[[which(names(ensemble.samples)!='soil')[1]]]$SLA) #HACK
-  }else{
-    load(file.path(settings$outdir, paste0("ensemble.samples.",settings$state.data.assimilation$prior,".Rdata")))
-    prior.sla <- ens.samples[[which(names(ensemble.samples)!='soil')[1]]]$SLA
-  }
+  prior.sla <- params[[which(names(ensemble.samples)!='soil')[1]]]$SLA
```

Comment:
Good catch @istfer -- we previously were loading ensemble.samples from file within the function but now we're passing them from params, so that should be names(params) now.